<launch>
  <!--args; can be passed from command line-->
  <arg name="parentFrame" default="/odom"/>
  <arg name="frame_name" default="/test"/> 
  <arg name="framesCsv"  default="$(find privacy)/config/frames/newThing.csv" />

  <!--Runs a makeFrame server that brings up an interactive marker and saves its placement. -->
  <node pkg="privacy" type="makeFrame.py" name="makeFrame">
    <!--the parent of the frame we're adding. -->
	<param name="parentFrame" value="$(arg parentFrame)"/>
  	<!--the name of the frame we're adding. -->
	<param name="frame_name" value="$(arg frame_name)"/>
  	<!-- the csv file the frame is being added to -->
	<param name="framesCsv" value="$(arg framesCsv)" />
  </node> 
</launch>  
